File: src/main/java/dev/debuggings/examplemod/commands/ConfigCommand.java
package dev.debuggings.examplemod.commands;

import dev.debuggings.examplemod.ExampleMod;
import gg.essential.api.EssentialAPI;
import gg.essential.api.commands.Command;
import gg.essential.api.commands.DefaultHandler;

public class ConfigCommand extends Command {
    public ConfigCommand() {
        super("examplemod");
    }

    @DefaultHandler
    public void handle() {
        EssentialAPI.getGuiUtil().openScreen(ExampleMod.config.gui());
    }
}

File: src/main/kotlin/dev/debuggings/examplemod/commands/ConfigCommand.kt
package dev.debuggings.examplemod.commands

import dev.debuggings.examplemod.core.Config
import gg.essential.api.EssentialAPI
import gg.essential.api.commands.Command
import gg.essential.api.commands.DefaultHandler

object ConfigCommand : Command("examplemod") {

    @DefaultHandler
    fun handle() {
        EssentialAPI.getGuiUtil().openScreen(Config.gui())
    }
}
